cloudify_cli/cli/cfy.py

```diff
@@ -167,10 +167,14 @@ def show_version(ctx, param, value):
     cli_version_data = {
         'version': pkg_resources.require('cloudify')[0].version
     }
-    rest_version_data = env.get_manager_version_data() \
-        if env.is_manager_active() else None
+    try:
+        rest_version_data = env.get_manager_version_data() \
+            if env.is_manager_active() else ""
+    except Exception as e:
+        rest_version_data = "Cloudify Manager unreachable. {}: {}\n".format(
+            type(e).__name__, str(e))
```

Comment:
Why not just output it to stderr at this point and exit with status 1?

Reply:
We need to output the CLI version in any case, so we can't exit quite yet here

Reply:
The problem is that this is then giving an exit status of 0 for a command that failed, and not putting the output in the right place (stderr for a problem).